src/components/TheUserMenu.vue

```diff
@@ -57,3 +67,11 @@ const menuItems = computed(() => {
 });
 
 </script>
+<style>
```

Comment:
Need to make the style scoped to prevent bleeding across components
```suggestion
<style scoped>
```